### PR TITLE
metrics: Fix and update metrics report

### LINKED ISSUE #3783: fix metrics report md

### PR DESCRIPTION
**Pull request by GabyCT**
This PR updates and fixes with the current metrics scripts that are
being supported on kata 2.0 the metrics_report.rmd

Fixes #3783

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

**Review by devimc — APPROVED** [review]

File: metrics/report/report_dockerfile/metrics_report.Rmd [hunk 1]
```diff
@@ -87,19 +87,6 @@ source('fio-writes.R')
 ```
 \pagebreak
 
-# Network CPU costs
```
Comment (Jakob-Naucke) [inline metrics/report/report_dockerfile/metrics_report.Rmd:90]: OOI why was this removed?

**Review by Jakob-Naucke — APPROVED** [review]

File: metrics/report/report_dockerfile/metrics_report.Rmd (hunk 1 above)
Comment (GabyCT) [reply metrics/report/report_dockerfile/metrics_report.Rmd:90]: @Jakob-Naucke currently we do not have this script running on the metrics CI
